packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -57,6 +57,12 @@ export interface IFluidContainer extends IEventProvider<IFluidContainerEvents> {
      */
     readonly connected: boolean;
 
+    /**
+     * Returns true if Fluid Container has data loss, container is closed at that same moment and
+     * there is no network connection to ordering service
```

Comment:
I would either expand this doc comment to include all the detail from the docs, or include a link to the docs with more information.